service/reserve_room_service.py

```diff
@@ -0,0 +1,48 @@
+from sqlalchemy_cockroachdb import run_transaction
```

Comment:
I have used https://github.com/ssw555tm012021Fall/halcyon-api/blob/main/service/room_service.py for service methods. Please use it if you think it's a good idea else it's fine